src/cogs/admin.py

```diff
@@ -13,42 +13,58 @@ def __init__(self, bot):
         self.bot = bot
         self._last_result = None
 
-        with open("src/backend/database.json", 'r') as file:
+        with open("src/backend/database.json", "r") as file:
             self.file = json.load(file)
 
     def cleanup_code(self, content):
         """Automatically removes code blocks from the code."""
+        content = (
+            content.replace("```", "\n```") # Prevents the last line being cut off
+            .replace("'", "\'") # ' -> \'
+            .replace('"', '\"') # " -> \"
+            .strip() # Remove trailing whitespace
+        )
+
         # remove ```py\n```
-        if content.startswith('```') and content.endswith('```'):
-            return '\n'.join(content.split('\n')[1:-1])
+        if content.startswith("```") and content.endswith("```"):
+            return "\n".join(content.split("\n")[1:-1])
 
         # remove `foo`
-        return content.strip('` \n')
+        return content.strip("` \n")
 
     @commands.command(hidden=True)
+    @commands.is_owner()
```

Comment:
Same here? im not sure if im owner even.

Reply:
Latest commit should fix this